admin.php

```diff
@@ -177,7 +177,7 @@ function wpcf7cf_after_save($contact_form) {
 function wpcf7cf_save_contact_form( $contact_form )
 {
 
-	if ( ! isset( $_POST ) || empty( $_POST ) || ! isset( $_POST['wpcf7cf-settings-text'] ) ) {
```

Comment:
Need to keep this for usage in end to end tests and when calling methods directly from command line.